core/standalone/Dockerfile

```diff
@@ -0,0 +1,33 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM amazoncorretto:8u232
```

Comment:
Shouldn‘t this use openjdk like all our other images?

Reply:
Which one is the recommended one to use?

Reply:
You can use `FROM scala`. This image is built by `common/scala` module and is used by other docker images like `controller` and `invoker`. Note this is based on debian

Reply:
I already did this. Works. Will submit the change with the rest.